src/layers/domain/repository/tests/test_device_repository.py

```diff
@@ -0,0 +1,57 @@
+from uuid import UUID
+
+import boto3
+import pytest
+from domain.core.accredited_system_id import AccreditedSystemId
+from domain.core.device import DeviceStatus, DeviceType, RelationshipType
+from domain.core.product_id import ProductId
+from domain.core.root import Root
+from domain.repository.device_repository import DeviceRepository
+
+from test_helpers.terraform import read_terraform_output
+
+
+@pytest.mark.wip
+@pytest.mark.integration
+def test__device_repository():
+    subject_id = UUID("774c8c06-0d15-4b7d-9c7e-4a358681b78b")
+    target_id = UUID("07bca737-1022-494c-8df0-5e7dd152ba59")
+    table_name = read_terraform_output("dynamodb_table_name.value")
+
+    org = Root.create_ods_organisation(ods_code="AB123")
+    team = org.create_product_team(
+        id=UUID("6f8c285e-04a2-4194-a84e-dabeba474ff7"), name="Team"
+    )
+    target = team.create_device(
```

Comment:
has this test been done pre code? I thought we were tackling devices this sprint haha

Reply:
the file got renamed but the old file still exists - I've removed the old file in the follow-up PR (https://github.com/NHSDigital/connecting-party-manager/pull/97/files)

Reply:
deleted here 
https://github.com/NHSDigital/connecting-party-manager/pull/97/files#diff-fcab149af365c4ec854d8a6aa4da6947ce3eb5b7981e49b099272c9faa83794c